bluto/doc.go

```diff
@@ -0,0 +1,6 @@
+/*Package bluto is connection pool wraper which can borrow redis connection from pool and return them when finished.
```

Comment:
> Package Bluto is a Redis connection pool which can lend Redis connections. The connection and the pool wrap and enhance their Redigo counterparts.

I think this is a better description as it doesn't involve in returning the connection and only in lending them.  (borrow is the reverse)
Also wraper => wrapper